server/nuxeo-nxr-server/src/main/resources/templates/common-base/client/scripts/deleteOrphanTasks.groovy

```diff
@@ -0,0 +1,37 @@
+import org.apache.logging.log4j.LogManager
+import org.nuxeo.ecm.platform.task.Task
+import org.nuxeo.ecm.core.api.IdRef
+import org.nuxeo.ecm.core.api.ScrollResult
+import org.nuxeo.runtime.transaction.TransactionHelper
+
+def log = LogManager.getLogger("deleteOrphanTasks.groovy")
+def nxql = "SELECT * FROM Document WHERE ecm:mixinType = 'Task'"
+def dryRun = true
+def session = Context.getCoreSession()
+def ScrollResult<String> scrollResult = session.scroll(nxql, 500, 60)
+def removedCount = 0
+def taskCount = 0
+
+log.info("Starting deleting orphan tasks...")
+while (scrollResult.hasResults()) {
+    taskCount += scrollResult.getResults().size()
+    def cache = [:]
```

Comment:
Should be outside the `while` as we may want to reuse the cache when scrolling.

Reply:
doh